storage/statedb/database.go

```diff
@@ -824,9 +825,10 @@ func (db *Database) Cap(limit common.StorageSize) error {
 	memcacheFlushSizeGauge.Update(int64(nodeSize - db.nodesSize))
 	memcacheFlushNodesGauge.Update(int64(nodes - len(db.nodes)))
 
-	logger.Info("Persisted nodes from memory database by Cap", "nodes", nodes-len(db.nodes), "size", nodeSize-db.nodesSize, "time", time.Since(start),
-		"flushnodes", db.flushnodes, "flushsize", db.flushsize, "flushtime", db.flushtime, "livenodes", len(db.nodes), "livesize", db.nodesSize)
-
+	logger.Info("Persisted nodes from memory database by Cap", "nodes", nodes-len(db.nodes),
+		"size", nodeSize-db.nodesSize, "preimagesSize", preimagesSize-db.preimagesSize, "time", time.Since(start),
```

Comment:
Just `preimagesSize` confuse me; it could be flushed preimageSize or remained preimageSize.
```suggestion
		"size", nodeSize-db.nodesSize, "flushedPreimagesSize", preimagesSize-db.preimagesSize, "time", time.Since(start),
```

Reply:
I followed the convention of other items; `node` means flushed node and `size` means flushed size. We may need to change the names of all items.

Reply:
@ehnuje How do you think about @winnie-byun 's comment?

Reply:
@aidan-kwon @winnie-byun I just noticed that this have logged the data from both the current and accumulated status.  `flushnodes` means the accumulated number of flushed nodes between `Database.Commit`, while `nodes` means the number of nodes flushed by the current `Cap` operation. 

If we want to add prefixes to the current data, as @winnie-byun suggested, then we also need to update the names of accumulated data. (e.g., `accumulatedFlushedNodes`). In my opinion, that would clarify the meaning but it will also makes the message verbose.